game-core/src/main/java/games/strategy/net/ServerMessenger.java

```diff
@@ -327,20 +322,16 @@ public void messageReceived(final MessageHeader msg, final SocketChannel channel
   }
 
   private void bareBonesSendChatMessage(final String message, final INode to) {
-    final List<Object> args = new ArrayList<>();
-    args.add(message);
-    final Class<? extends Object>[] argTypes = new Class<?>[1];
-    argTypes[0] = args.get(0).getClass();
-    final RemoteName rn;
-    if (isLobby()) {
-      rn = new RemoteName(ChatController.getChatChannelName("_LOBBY_CHAT"), IChatChannel.class);
-    } else {
-      rn = new RemoteName(
-          ChatController.getChatChannelName("games.strategy.engine.framework.ui.ServerStartup.CHAT_NAME"),
-          IChatChannel.class);
-    }
-    final RemoteMethodCall call =
-        new RemoteMethodCall(rn.getName(), "chatOccured", args.toArray(), argTypes, rn.getClazz());
+    final RemoteName rn = new RemoteName(isLobby()
+        ? ChatController.getChatChannelName("_LOBBY_CHAT")
+        : ChatController.getChatChannelName("games.strategy.engine.framework.ui.ServerStartup.CHAT_NAME"),
+        IChatChannel.class);
+    final RemoteMethodCall call = new RemoteMethodCall(
+        rn.getName(),
+        "chatOccured",
+        new String[] {message},
```

Comment:
Would it be more appropriate to create a new `Object[]` here instead of `String[]`?  I don't think anything will break in the current code, but just in case `RemoteMethodCall` ever tries to do something funky with this array like assign a non-`String` to an element, we'll avoid a potential runtime error.